zhaquirks/lixee/zlinky.py

```diff
@@ -13,7 +15,115 @@
     OUTPUT_CLUSTERS,
     PROFILE_ID,
 )
-from zhaquirks.lixee import LIXEE
+from zhaquirks.lixee import LIXEE, ZLINKY_MANUFACTURER_CLUSTER_ID
+
+
+class ZLinkyTICManufacturerCluster(CustomCluster, ManufacturerSpecificCluster):
+    """ZLinkyTICManufacturerCluster manufacturer cluster."""
+
+    cluster_id = ZLINKY_MANUFACTURER_CLUSTER_ID
+    name = "ZLinky_TIC Manufacturer specific"
+    ep_attribute = "zlinky_manufacturer_specific"
+
+    # The attribute comments below are in French to match the reference documentation,
+    # see https://github.com/fairecasoimeme/Zlinky_TIC/tree/v9.0#synth%C3%A8se-d%C3%A9veloppeur
+    # and https://github.com/fairecasoimeme/Zlinky_TIC/blob/v9.0/ZLinky/Source/LixeeCluster.h
+    attributes = {
+        # Historical mode: OPTARIF "Option tarifaire" / String 4 car
+        # Standard mode: NGTF "Nom du calendrier tarifaire fournisseur" / String 16 car
+        0x0000: (
+            "hist_tariff_option_or_std_supplier_price_schedule_name",
+            t.LimitedCharString(16),
+            True,
+        ),
```

Comment:
It would be awesome to have descriptions on attributes that would show up in the UI, e.g.:
```suggestion
        0x0000: (
            "hist_tariff_option_or_std_supplier_price_schedule_name",
            t.LimitedCharString(16),
            True,
            "Historical mode: Option tarifaire / Standard mode: Nom du calendrier tarifaire fournisseur",
        ),
```

Reply:
@puddly whatcha think about this request?